src/components/RichTextEditor/utils/getTextStyles.ts

```diff
@@ -1,14 +1,13 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Maybe rename it to something like `textStyles` as getTextStyle function is removed